packages/atomic/cypress/integration-insights-panel/insights-panel.cypress.ts

```diff
@@ -164,6 +164,7 @@ describe('Insights panel test suites', () => {
         .shadow()
         .find('input')
         .type('test{enter}');
+      cy.wait(200);
```

Comment:
I'm a bit confused as to why this is necessary, since in theory the next `it` should be waiting for `querySummary()` to exist.

If possible, I prefer to rely on Cypress's retry mechanism.

Reply:
It triggers a search which updates the UI, same reason why we have these wait everywhere in the Search cypress tests